setup.py

```diff
@@ -13,9 +13,8 @@ def find_version(*paths):
     version_file_path = os.path.join(ROOT_DIR, *paths)
     with open(version_file_path) as file_stream:
         version_match = re.search(
-            r"^__version__ = ['\"]([^'\"]*)['\"]",
-            file_stream.read(),
-            re.M)
+            r"^__version__ = ['\"]([^'\"]*)['\"]", file_stream.read(), re.M
```

Comment:
Minor: Going back to the discussion on diff-friendly code, why consolidate all 3 args on one-line instead of one arg per line? Did the current linter config do this automatically (in which case, assuming it's 100% consistent with Ray, maybe we could recommend a change to the linter config for the main Ray repo)?

Reply:
Hmmm... was it because of the last line ending in `re.M)`? I wonder if it would have done the same if it was instead split into two lines with a trailing comma like:
```
    re.M,
)
```

Either way, I guess this is one case where I don't feel like the current config is necessarily improving code, but that could just be me.